news/2025/2025-01-23-osumania-7k-world-cup-2025-round-of-32-recap.md

```diff
@@ -0,0 +1,186 @@
+
+But that loss did not discourage Spain, as they quickly got back on their feet, winning map after map. By winning their break point, Spain managed to reach a lead of 4-1. With a clever used of what could very well be their last pick, Sweden chose [AAAA + yadrigg - Sakase Natsuzora, Koi no Hana.](https://osu.ppy.sh/beatmapsets/2307438#mania/4934949) (LN3) to try and gain some ground. It was not an easy task as Spain's accuracy far surpassed their opponents', but thanks to ::{ flag=ES }:: [Manuel-san_UWU](https://osu.ppy.sh/users/23244116)'s series of misses, the balance quickly shifted towards Sweden who managed to stay afloat, managing to overturn Spain's counter pick.
+
+With one point short of a tie, Sweden took a risky gamble and chose Spain's protect, [Tedjimo yomigY - Usuva Kagerow](https://osu.ppy.sh/beatmapsets/2307475#mania/4935023) (HB1). A precarious gambit that would end up sealing this match's fate, but not without giving Spain a run for their money! Were it not be for ::{ flag=ES }:: [Luqas678](https://osu.ppy.sh/users/26688450)'s impressive full combo, we may've been recapping the tiebreaker between our two teams.
+
+With all of that said, we believe this match was a testament on how important it is to play with your strengths and against your opponents' weaknesses. Both teams held an edge over the other's blind spots. A match that was not only fought through technical skill alone, but won through strategy and thoughtful choices.
+
+### ::{ flag=FR }:: France vs. ::{ flag=MO }:: Macau
+
+**[You can view the match information here!](https://osu.ppy.sh/community/matches/116869938)**\
+**[You can view the match VOD here!](https://www.twitch.tv/videos/2357648754)**
+
+One of our most intersting matches this weekend must've been between ::{ flag=FR }:: France (#15) and ::{ flag=MO }:: Macau (#18). While France has had a consistent presence in the tournament for many years now, this is the first time since 2019 that Macau has made it to the elimination phase. A considerable feat all things considered, if we take into account the fact that Macau only disposed of two players, as opposed to France's more robust rooster of four. An unusual match to experience for sure.
```

Comment:
```suggestion
One of our most interesting matches this weekend must've been between ::{ flag=FR }:: France (#15) and ::{ flag=MO }:: Macau (#18). While France has had a consistent presence in the tournament for many years now, this is the first time since 2019 that Macau has made it to the elimination phase. A considerable feat all things considered, if we take into account the fact that Macau only disposed of two players, as opposed to France's more robust roster of four. An unusual match to experience for sure.
```